CONTRIBUTING.md

```diff
@@ -12,16 +12,19 @@ All issues reported to Djang Dramatiq MUST follow the issue template. Any issues
 
 ## Running the Project Locally
 
-1. `git clone` the project.
-2. Create a virtual environment and install the dev dependencies with `pip install -e '.[dev]'`.
-3. Run tests with `python -m pytest`.
+1. `git clone` the project
+2. Use [uv](https://uv.astral.sh/) to install dependencies:
+   ```bash
+   uv sync --group dev
+   ```
+3. Run tests with `uv run pytest`
```

Comment:
In theory in our setup here you won't even need step 2.

If you have uv installed and you run uv run pytest, it will check out pyproject.toml, install dependencies from default groups ([which `dev` is in](https://docs.astral.sh/uv/concepts/projects/dependencies/#default-groups)) and will just run tests.